src/Handlers/PaymentResponseHandler.php

```diff
@@ -26,14 +26,18 @@
 
 namespace Adyen\Shopware\Handlers;
 
-use Adyen\AdyenException;
 use Psr\Log\LoggerInterface;
 use Adyen\Shopware\Service\PaymentResponseService;
+use Shopware\Core\Checkout\Order\Aggregate\OrderTransaction\OrderTransactionStateHandler;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Shopware\Core\Checkout\Payment\Cart\AsyncPaymentTransactionStruct;
+use Shopware\Core\Checkout\Payment\Exception\AsyncPaymentProcessException;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Symfony\Component\HttpFoundation\RedirectResponse;
 
 class PaymentResponseHandler
 {
-    const ADYEN_MERCHANT_REFERENCE = 'adyenMerchantReference';
+    const ADYEN_MERCHANT_REFERENCE = 'merchantReference';
```

Comment:
This must match the GET var the is returned in the URL, in PS is adyenMerchantReference. Did you change it for some different reason?

Reply:
aaaa sorry so this is for the GET parameter - I use it for the synch response merchantReference var
I will create a new one!